README.md

```diff
@@ -83,14 +83,21 @@ $ wrangler publish
 
 ### 🔧 `config`
 
-  Configure your global Cloudflare user. This is an interactive command that will prompt you for your email and API key:
+  Configure your global Cloudflare user. This is an interactive command that will prompt you for your API token:
 
   ```bash
   wrangler config
+  Enter API token:
+  superlongapitoken
+  ```
+
+  You can also provide your email and global API key (this is not recommended for security reasons):
```

Comment:
nit: should we link here to the same resource we link to in the terminal warning?

Reply:
can do, easy fix :)